packages/react-mongo/react-mongo.ts

```diff
@@ -0,0 +1,118 @@
+import { Meteor } from 'meteor/meteor'
+import { Mongo } from 'meteor/mongo'
+import { Tracker } from 'meteor/tracker'
+import { EJSON } from 'meteor/ejson'
+import { useEffect, useMemo, useReducer, useRef, DependencyList } from 'react'
+
+const fur = (x: number): number => x + 1
+const useForceUpdate = () => useReducer(fur, 0)[1]
+
+type useSubscriptionRefs = {
+  facade: Meteor.SubscriptionHandle,
+  subscription?: Meteor.SubscriptionHandle,
+  computation?: Tracker.Computation,
+  updateOnReady: boolean,
+  isReady: boolean,
+  params: {
+    name?: string,
+    args: any[]
+  }
+}
+
+const useSubscriptionClient = (name?: string, ...args: any[]) => {
+  const forceUpdate = useForceUpdate()
+
+  const ref = useRef<useSubscriptionRefs>({
+    facade: {
+      stop () {
+        refs.subscription?.stop()
```

Comment:
Yes! Will fix.

Reply:
Actually, I was incorrect. This is supposed to stop the subscription. I'm not aware of cases where you'd want to do that, but this method is exposed on the Meteor.SubscriptionHandle, so I exposed it here in case someone has a use case for it.